### PR TITLE
Add EncryptionMethod tests

### PR DESCRIPTION
**Pull request by hughbe**
Contributes to #16829

Increases CC to 100% and also tests some more utils helpers

@krwq

### CODE REVIEW

File: src/System.Security.Cryptography.Xml/tests/EncryptionMethodTests.cs [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            Assert.Equal(value, method.KeySize);
+        }
+
+        [Theory]
+        [InlineData(-1)]
+        [InlineData(0)]
+        public void KeySize_SetNegativeValue_ThrowsArgumentOutOfRangeException(int value)
+        {
+            EncryptionMethod method = new EncryptionMethod();
+            Assert.Throws<ArgumentOutOfRangeException>("The key size should be a non negative integer.", () => method.KeySize = value);
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Xml/tests/EncryptionMethodTests.cs]: This is a really long line, and the message verification is only correct when using the en-us resources.

Though, I think that's supposed to be the parameter name, meaning we have a buggy exception :smile:.

**Comment by bartonjs** [comment]
@dotnet-bot test code coverage please

**Comment by krwq** [comment]
@dotnet-bot test code coverage please

**Comment by hughbe** [comment]
I've fixed the param name - looks like we only run netcoreapp tests against System.Security.Cryptography.Xml so we don't need to nerf the tests against netfx

**Pull request merged**